docs/how-to-guides/basic_image.rst

```diff
@@ -0,0 +1,99 @@
+.. _howto_buld_basic_image:
+
+===================
+Build a basic image
+===================
+
+These instructions describe how to build a ``classic`` server 24.04 AMD64 Ubuntu image
+with Imagecraft.
+
+Prerequisites
+-------------
+
+- AMD64 machine with Ubuntu 24.04
+- ``snapd`` installed
+- 15GiB or more disk space to process the build and hold the resulting image
+
+.. note:: Following instructions build an **AMD64** image on an **AMD64** machine.
+          Building on another architecture would need several modifications not
+          described on this page.
+
+Install Imagecraft
+~~~~~~~~~~~~~~~~~~
+
+Imagecraft is only available as a snap in the ``latest/edge`` channel
+`from the Snapstore <https://snapcraft.io/imagecraft>`_. Install it with:
+
+.. code-block::
+
+    sudo snap install --classic --edge imagecraft
+
+Verify that ``imagecraft`` is properly installed:
+
+.. code-block::
+
+    imagecraft
+
+.. important:: ``imagecraft`` requires **elevated permissions**. Run it with **root**
+               privileges or using ``sudo`` if you plan on using the executing system
+               as the build system ("destructive mode").
+
+
+Prepare the configuration
+-------------------------
+
+Imagecraft configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Save the following content as ``imagecraft.yaml``:
+
+.. literalinclude:: code/basic_imagecraft.yaml
+    :caption: imagecraft.yaml
+    :language: yaml
+
+.. note:: The name of the configuration file, ``imagecraft.yaml``, is **important**
```

Comment:
Instead of making this a note after the reader copies the code, what do you think about making it a regular paragraph at the start of the section? I think it would be better if the user got this information before they took action.